utils.go

```diff
@@ -98,3 +98,31 @@ func WriteFile(fs Filesystem, filename string, data []byte, perm os.FileMode) er
 
 	return err
 }
+
+// CopyPath copies file across filesystems.
```

Comment:
To be consistent with the rest of the utils, maybe this should be called `CopyFile`, although I would call it simply `Copy`.